.env.example

```diff
@@ -110,6 +110,9 @@ GITHUB_TOKEN=
 
 
 ## Jenkins, optional, for triggering Jenkins builds after deployment
+## When JENKINS_NOTIFY_COMMITTERS is set, it will send out emails of deployer, buddy-approver ( if available ), and committers
+## to jenkins as "emails" param. If GOOGLE_DOMAIN is set, then emails are filtered by domain.
 # JENKINS_URL= # server_url of jenkins
 # JENKINS_USERNAME= # user id
 # JENKINS_API_KEY= # API Token from user / Configure page
+# JENKINS_NOTIFY_COMMITTERS=1 #optional
```

Comment:
I think the committer notification will nice in automated stages (master)
... but could be annoying when going to production (lots of authors) ... so
per-stage kind of makes sense ... but could also not have notifications for
the production stages ...

for classic I'd prefer to have it on master, but not on staging/prod ...
other projects might vary ... adding a checkbox also means you don't have
to go through ops :D

On Thu, Aug 18, 2016 at 9:38 PM, Zehua Liu <notifications@github.com> wrote:

> In .env.example
> <https://github.com/zendesk/samson/pull/1214#discussion_r75426681>:
>
> >  # JENKINS_URL= # server_url of jenkins
> >  # JENKINS_USERNAME= # user id
> >  # JENKINS_API_KEY= # API Token from user / Configure page
> > +# JENKINS_NOTIFY_COMMITTERS=1 #optional
>
> @grosser <https://github.com/grosser> that's related one of the two
> questions that we raised in the description:
>
> Should there be corresponding UI changes for sending emails of committers.
> Presently, if JENKINS_NOTIFY_COMMITTERS is set, it will send email of
> committers to jenkins, otherwise not.
>
> Do we want to make this as a checkbox on the staging configuration page so
> that owners of different stages could decide whether they want to notify
> committers and/or deployer/buddy? Or are we good with starting with a
> global option.
>
> Of course, it's still up to the jenkins builds to configure whether this
> new emails build parameter will be used to send email notifications.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/zendesk/samson/pull/1214/files/ac0111ac4d25e0062416c4302b0ab65995ee31bd#r75426681>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAAsZ-HX5BdHuLaD7EbXMTfqOH-pr-3Sks5qhTNfgaJpZM4JmQsW>
> .
>